check_stability.py

```diff
@@ -435,6 +435,20 @@ def get_affected_testfiles(files_changed):
         repo_path = "/" + os.path.relpath(full_path, wpt_root).replace(os.path.sep, "/")
         nontest_changed_paths.add((full_path, repo_path))
 
+    # JavaScript files named with a `.any.js` suffix are interpreted in a
+    # number of different contexts, but they are all interpreted using a
+    # document that does not have without a corresponding file on disk. Changes
+    # to these test files should trigger stability checking using those
+    # "virtual" documents.
+    multiglobal_re = re.compile(r'^(.*)\.any\.js$')
```

Comment:
We have access to the manifest, right? So we should probably read the manifest to get a map of path:tests rather than recreating some of the logic here.